governance/meetings/2023.md

```diff
@@ -0,0 +1,1235 @@
+      * [https://github.com/ossf/scorecard-webapp/tree/main/scorecards-site/static/viewer](https://github.com/ossf/scorecard-webapp/tree/main/scorecards-site/static/viewer)    
+    *   
+* Vendor news  
+  * Officially started last week, roadmap defined and approved.  
+    * [Scorecard Community Roadmap](https://docs.google.com/document/d/1ttLH1U4BRZkgVukBltLxKyP7UmFSLIyPtUUe1G86Sv0/edit#heading=h.ou02ukhsja30)  
+  * Working on moving checks structured results?  
+  * Also, false positives and negatives  
+* 
+
+## Aug 24, 2023
+
+Attendees:
+
+* David A. Wheeler (LF)  
+* Ian Dunbar-Hall (Lockheed Martin)  
+* Raghav Kaul (Google)  
+* Evan Anderson (Stacklok)
+
+Agenda:
+
+* GitLab Namespace Squatting  
+  * [https://gitlab.com/ossf](https://gitlab.com/ossf)  
+  * [https://gitlab.com/openssf](https://gitlab.com/openssf)  
+* GitHub Namespace squatting recently resolved (it’s why we’ve been using “ossf” organization)  
+  * Slack conversation \- [https://openssf.slack.com/archives/C05BETTLHEE/p1690586545571689](https://openssf.slack.com/archives/C05BETTLHEE/p1690586545571689)   
+  * Sam White is GitLab product manager for “security product line”. We asked about GitLab squatting.  
+    * [https://openssf.slack.com/archives/C052Z3A3FQA](https://openssf.slack.com/archives/C052Z3A3FQA)  
+* The blog post announcing Scorecard 4.12, including GitLab support, is basically done. Need a bio from Keith Ganger & Jim Robison. Ian is trying to reach Keith & Jim now.  
+* 
+
+Actions:
+
+- Ian Dunbar-Hall \- provide Sam White contact info to David Wheeler. David’s email: dwheeler @ linuxfoundation . org  
+- Ian Dunbar-Hall \- track down keith and jim’s bios for blog post.
+
+## Aug 21, 2023
+
+Attendees:
+
+* Dan Appelquist (Snyk)  
+* David A. Wheeler (LF)  
+* Adrianne Marcum (LF)  
+* Cheuk Ho (LF)
+
+Agenda:
+
+*  Announcements  
+  * Vendor contract process underway for [work outlined in the roadmap](https://docs.google.com/document/d/1ttLH1U4BRZkgVukBltLxKyP7UmFSLIyPtUUe1G86Sv0/edit#heading=h.ou02ukhsja30), we’ve interviewed potential contractors  
+*  [https://github.com/ossf/scorecard/pull/3107](https://github.com/ossf/scorecard/pull/3107)  
+  * All resolved, we just need to make sure the checks all pass  
+  * David W. to merge once the checks run
+
+## Aug 10, 2023
+
+Attendees:
+
+* Jeff Mendoza (Kusari)  
+* Spencer Schrock (Google)  
+* Pedro Nacht (Google)  
+* Diogo Teles (Google)  
+* David A. Wheeler (LF)  
+* Adrianne Marcum (LF)  
+* Cheuk Ho (LF)  
+* Evan Anderson (Stacklok)
+
+Agenda:
+
+*  Announcements  
+  * New meeting link\! Should be updated on the calendar. More details in slack [\[link\]](https://openssf.slack.com/archives/C019M98JSHK/p1691174555715859)  
+    * \[Evan\] calendar seems to still have old link  
+    * \[Evan\] Need LFX account to join  
+    * [Join Link](#meeting-info)  
```

Comment:
I did a quick search for other local links. We can probably ignore this one? or unlink/delete the bullet?

Reply:
Fixed.